services/status_server/status_server.py

```diff
@@ -0,0 +1,22 @@
+import tornado.ioloop
+import tornado.web
+
+from baselayer.app.env import load_env
+from baselayer.log import make_log
+
+env, cfg = load_env()
+
+
+class MainHandler(tornado.web.RequestHandler):
+    def get(self):
+        self.write("<h2>SkyPortal is being provisioned</h2>")
```

Comment:
So this always shows the same thing?

Reply:
Yes, it's just a backup so that we don't see the 5xx errors.  Is there something else that would be useful to show?

Reply:
(Note that this is the backup server, so it will only be hit until SP itself is ready.)